src/EPR.Calculator.API/Exporter/CalcResultsExporter.cs

```diff
@@ -9,7 +8,7 @@ namespace EPR.Calculator.API.Exporter
 {
     public class CalcResultsExporter : ICalcResultsExporter<CalcResult>
     {
-        private readonly IBlobStorageService _blobStorageService;
+        private readonly IStorageService _iStorageService;
```

Comment:
Can __iStorageService_ be changed to __storageService_ to match the naming convention?